modules/microshift-auto-apply-manifests.adoc

```diff
@@ -50,7 +50,8 @@ $ sudo mkdir -p ${MANIFEST_DIR}
 +
 [source,terminal]
 ----
-$ sudo tee ${MANIFEST_DIR}/busybox.yaml &>/dev/null <<EOF
+$ sudo tee ${MANIFEST_DIR}/busybox.yaml &>/dev/null<<
+  EOF
```

Comment:
Line 54 should move down below what is now line 79 to close out the command. The `<<EOF` starts a "here document" and the shell looks for the `EOF` portion to know when the input should be considered "done".